webpack/assets/javascripts/react_app/components/common/AutoComplete/AutoComplete.stories.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-alert */
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Row, Col } from 'react-bootstrap';
```

Comment:
Row and Col can be imported from `patternfly-react`
I think you can also import `Grid` and do `Grid.Row` `Grid.Col`